pkgs/servers/jigasi/sip-communicator.properties

```diff
@@ -0,0 +1,252 @@
+#Sample config with one XMPP and one SIP account configured
```

Comment:
Does this file need to be committed in nixpkgs? It looks like it could just be copied from `$src` to `$out`.

Reply:
Good catch! Will update it shortly.